diagnostic_aggregator/test/add_analyzers_test.py

```diff
@@ -122,8 +123,8 @@ def test_add_agg(self):
             agg_paths = [msg.name for name, msg in self.agg_msgs.iteritems()]
             self.assert_(all(expected in agg_paths for expected in self.expected))
                 
+        resp = self.add_diagnostics(load_namespace=self.namespace)
 
-        self.bond.shutdown()
```

Comment:
Is it intentional to remove the bond shutdown here ?
I would expect that having it or not would probably make a difference...